website/docs/r/evidently_launch.html.markdown

```diff
@@ -0,0 +1,349 @@
+* `scheduled_splits_config` - (Optional) A block that defines the traffic allocation percentages among the feature variations during each step of the launch. [Detailed below](#scheduled_splits_config).
+* `tags` - (Optional) Tags to apply to the launch. If configured with a provider [`default_tags` configuration block](/docs/providers/aws/index.html#default_tags-configuration-block) present, tags with matching keys will overwrite those defined at the provider-level.
+
+### `groups`
+
+The `groups` block supports the following arguments:
+
+* `description` - (Optional) Specifies the description of the launch group.
+* `feature` - (Required) Specifies the name of the feature that the launch is using.
+* `name` - (Required) Specifies the name of the lahnch group.
+* `variation` - (Required) Specifies the feature variation to use for this launch group.
+
+### `metric_monitors`
+
+The `metric_monitors` block supports the following arguments:
+
+* `metric_definition` - (Required) A block that defines the metric. [Detailed below](#metric_definition).
+
+#### `metric_definition`
+
+The `metric_definition` block supports the following arguments:
+
+* `entity_id_key` - (Required) Specifies the entity, such as a user or session, that does an action that causes a metric value to be recorded. An example is `userDetails.userID`.
+* `event_pattern` - (Required) Specifies The EventBridge event pattern that defines how the metric is recorded.
+* `name` - (Required) Specifies the name for the metric.
+* `unit_label` - (Optional) Specifies a label for the units that the metric is measuring.
+* `value_key` - (Required) Specifies the value that is tracked to produce the metric.
+
+### `scheduled_splits_config`
+
+The `scheduled_splits_config` block supports the following arguments:
+
+* `steps` - (Required) One or up to six blocks that define the traffic allocation percentages among the feature variations during each step of the launch. This also defines the start time of each step. [Detailed below](#steps).
+
+#### `steps`
+
+The `steps` block supports the following arguments:
+
+* `group_weights` - (Required) The traffic allocation percentages among the feature variations during one step of a launch. This is a set of key-value pairs. The keys are variation names. The values represent the percentage of traffic to allocate to that variation during this step. For more information, refer to the [AWS documentation for ScheduledSplitConfig groupWeights](https://docs.aws.amazon.com/cloudwatchevidently/latest/APIReference/API_ScheduledSplitConfig.html).
+* `segment_overrides` - (Required) One or up to six blocks that specify different traffic splits for one or more audience segments. A segment is a portion of your audience that share one or more characteristics. Examples could be Chrome browser users, users in Europe, or Firefox browser users in Europe who also fit other criteria that your application collects, such as age. [Detailed below](#segment_overrides).
```

Comment:
```suggestion
* `segment_overrides` - (Required) One or up to six blocks that specify different traffic splits for one or more audience segments. A segment is a portion of your audience that share one or more characteristics. Examples could be Chrome browser users, users in Europe, or Firefox browser users in Europe who also fit other criteria that your application collects, such as age. [Detailed below](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/evidently_launch#segment_overrides).
```

Reply:
Following the Kendra docs https://github.com/hashicorp/terraform-provider-aws/blob/fcb0d1ed516f6036c145db8f1ea76709c2298382/website/docs/r/kendra_experience.html.markdown?plain=1#L49 full length URLs are not required for linking, pretty cool stuff